code/src/App.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './Album'
+import { Header } from './Header'
 
 console.log(data)
+console.log(Album)
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+      <Header />
+        <section id='album-container'>
```

Comment:
How come you are writing `'album-container'` and not `"album-container"` here? On other places you are writing with `"`, so it would be good to write that everywhere